src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1723,9 +1723,21 @@ export class ResultsViewPageStore {
             } else {
                 // would be nice to have an endpoint that would return multiple sample lists
                 // but this will only ever happen one for each study selected (and in queries where a sample list is specified)
-                const allSampleLists = await Promise.all(this.rvQuery.samplesSpecification.map((spec) => {
+                let samplesSpecifications = [];
+                // get sample specifications from physical studies if we are querying virtual study
+                if (this.queriedVirtualStudies.result!.length > 0){
```

Comment:
can you verify that this works is there's a mix of virtual studies and physical studies in query?